### PR TITLE
gh-38: Update update_switcher_chart with title and line plot

### LINKED ISSUE #38: Chart "Postęp tłumaczenia" in ReadMe needs clarification
Chart suggests total translation and not % of the goal.

### PR DESCRIPTION
**Pull request by StanFromIreland**
language-switcher-progress

As per my ideas in #38 In my opinion a line plot looks much clearer and easier to read. We should also look into updating the code as the plot_date function was deprecated in Matplotlib 3.9 and will be removed in 3.11.

Closes #38.

### CODE REVIEW

File: update_switcher_chart.py [hunk 1]
```diff
@@ -24,9 +24,10 @@
     progress.append(number)
     dates.append(datetime.fromtimestamp(commit.authored_date))
 
-pyplot.plot_date(dates, progress)
+pyplot.plot_date(dates, progress, linestyle='-',marker='')
```
Comment (m-aciek) [inline update_switcher_chart.py]:
```suggestion
pyplot.plot_date(dates, progress, linestyle='-', marker='')
```

File: update_switcher_chart.py [hunk 2]
```diff
@@ -24,9 +24,10 @@
...
 pyplot.ylim(ymin=0)
 pyplot.grid()
 pyplot.gcf().autofmt_xdate()
 pyplot.gca().yaxis.set_major_formatter(PercentFormatter())
+pyplot.title("Postęp tłumaczenia do dodatku przełącznika języków")
```
Comment (m-aciek) [inline update_switcher_chart.py]:
```suggestion
pyplot.title("Postęp tłumaczenia do dodania do przełącznika języków")
```

**Review by m-aciek — COMMENTED** [review]
I suggest minor improvements

**Comment by StanFromIreland** [comment]
All good!

**Comment by m-aciek** [comment]
Thanks!

**Pull request merged**